docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -1283,6 +1392,17 @@ builder.dynamicHeader("Authorization", tokenUni);
 VertxDynamicGraphQLClient client = builder.build();
 ----
 
+[[configuration-reference]]
+== Configuration Reference
+
+=== OIDC Client
+
+include::{generated-dir}/config/quarkus-oidc-client.adoc[opts=optional, leveloffset=+1]
+
+=== OIDC Token Propagation
```

Comment:
@sheilamjones I was wondering if it should be `OIDC token propagation` (and similarly, `OIDC client` just above) ? This is the main reason why I've asked to review :-)

Reply:
Yes, you are correct. For titles, they should be in sentence case, that is "OIDC token propagation" and also "OIDC client".